pilot/helpers/cli.py

```diff
@@ -473,6 +473,8 @@ def run_command_until_success(convo, command,
         return {'success': False, 'user_input': response}
 
     response = check_if_command_successful(convo, command, cli_response, response, exit_code, additional_message)
+    if response:
```

Comment:
Unrelated to telemetry, but useful to avoid going into debugging for whitespace reasons.